benefits/eligibility/templates/eligibility/unverified.html

```diff
@@ -16,19 +16,18 @@
 {% endblock icon %}
 
 {% block headline-text %}
-  {% translate "Your eligibility could not be verified." %}
```

Comment:
I believe we still need this copy and the "That's okay!... " copy for when a user is not eligible for a non-agency card flow.

Figma:
![image](https://github.com/user-attachments/assets/20b822c1-75b7-47dc-bdd5-8c53bee0b667)

https://www.figma.com/design/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?node-id=9234-110446&m=dev

Reply:
Sidenote: there's an outdated screen in Figma... right below those "Login" and "Non-Login" screens, it's showing that agency cards are supposed to have the old screen. But higher up in the file, it shows the new screen.

![image](https://github.com/user-attachments/assets/64c29828-9484-4502-81ac-b4c9b208588f)

See https://github.com/cal-itp/benefits/issues/2013 and https://github.com/cal-itp/benefits/pull/2053 where the new screen replaced the old one for agency cards.

Reply:
Ahh thank you for this clairification! Will fix.

Reply:
Okay I fixed it with a new approach, that I hope is more clear:
- `unverified.html` is for the Login flow
- `unverified-error-agency-base` is for all agency card flows
Updated Spanish for the `unverified` page.